beta/project-griffon/set-up-project-griffon.md

```diff
@@ -118,13 +122,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 
 ### Start a Project Griffon session
 
-After the extension has been registered, start a Project Griffon session by using the following API:
+1. [Create](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon#1-create-a-session) a new session in the [griffon UI](https://griffon.adobe.com/) by providing a base url for your application.
+
+2. Configure your iOS/Android application to handle the griffon session deeplink obtained from the griffon.
 
 {% tabs %}
 {% tab title="Android" %}
+
+Once your app is configured to handle the deeplink. You can start the griffon session by opening the deeplink provided from the griffon UI
+![](../../.gitbook/assets/griffon_deeplink_configure.png)
+
 ### startSession
 
-This API accepts a deep link to begin a session. After this API is called, to begin a session, the SDK displays a PIN authentication overlay on your app.
+Alternatively, startSession API can be used to start a griffon session by providing the deeplink. After this API is called, to begin a session, the SDK displays a PIN authentication overlay on your app.
```

Comment:
Please reword the 1st sentence:
"You can also start a Griffon session by providing the deeplink and using `startSession`.